### PR TITLE
Update mageck mle module pytest

### PR DESCRIPTION
**Pull request by LaurenceKuhl**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by LaurenceKuhl** [comment]
@nf-core-bot fix linting

File: modules/nf-core/mageck/mle/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        when {
+            params {
+                // define parameters here. Example:
+                // outdir = "tests/results"
+            }
+            process {
+                """
+                // define inputs of the process here. Example:
+                // input[0] = file("test-file.txt")
```
Comment (sateeshperi) [inline modules/nf-core/mageck/mle/main.nf.test:17]: is this test file complete ?

File: modules/nf-core/mageck/mle/tests/main.nf.test.snap [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "mageck_mle - csv": {
+        "content": [
+            [
+                "versions.yml:md5,95ca7c793029a1103d38cabff209b874"
+            ]
```
Comment (sateeshperi) [inline modules/nf-core/mageck/mle/tests/main.nf.test.snap:6]: only versions is captured in the snap...what abt the rest of the output ?

File: modules/nf-core/mageck/mle/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// nf-core modules test mageck/mle
```
Comment (sateeshperi) [inline modules/nf-core/mageck/mle/tests/main.nf.test:1]:
```suggestion
```

File: modules/nf-core/mageck/mle/tests/main.nf.test [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                    checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match() },
+                { assert file(process.out.gene_summary).readLines().first().contains("Gene")}
```
Comment (sateeshperi) [inline modules/nf-core/mageck/mle/tests/main.nf.test:32]:
```suggestion
                { assert file(process.out.gene_summary).readLines().first().contains("Gene")},
```
missing a comma at the end of line

File: modules/nf-core/mageck/mle/tests/main.nf.test [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match() },
+                { assert snapshot(
+                    process.out.gene_summary,
+                    process.out.sgrna_summary       )
+                }
```
Comment (sateeshperi) [inline modules/nf-core/mageck/mle/tests/main.nf.test:65]: the stub assertions can be similar to the test above

**Review by nvnieuwk — APPROVED** [review]
LGTM

**Pull request merged**